domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/stops/LocationBase.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.stops;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Base class for all entities defined in stops.txt
+ */
+public abstract class LocationBase {
```

Comment:
Should LocationBase derive from a generic GtfsEntity type? Since we want to add stop objects, agency objects etc to a GtfsDataRepository. That could ease the process of type resolution.

Reply:
I think it makes sense to have a `GtfsEntity` superclass or interface if there is a common implementation or interface for all objects. And the only fields I can think of that would make sense is an ID for the GTFS dataset (maybe a hash of the file?), a globally unique ID for the entity (something like [Transitland's Onestop ID](https://transit.land/documentation/onestop-id-scheme/)), or both.